.github/workflows/release-code-freeze.yml

```diff
@@ -75,6 +75,8 @@ jobs:
     uses: ./.github/workflows/release-pr.yml
     with:
       releaseVersion: ${{ needs.check-code-freeze.outputs.nextReleaseVersion }}
+    secrets:
+      BOTWOO_TOKEN: ${{ secrets.BOTWOO_TOKEN }}
```

Comment:
Would we have any unintended side effects if we just `inherit` all secrets to the nested workflow? It'd be handy otherwise to avoid adding these individually for future scenarios we might miss.

Reply:
I wouldn't expect side effects but I think the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege) is always good to follow.
That said, I don't have a particularly strong opinion on this

Reply:
I agree in general, but in this case, it's easy to miss adding a new secret here, just like we missed, and then could be tricky to debug. 

So we should either add a comment here mentioning that one should add an entry here for it to be inherited in nested workflows, or `inherit` all to avoid the same. I'll let you decide which way to go. 😄

Reply:
Made the nested workflow inherit all

Test run: https://github.com/Automattic/woocommerce-payments/actions/runs/3730484568